ModernWpf/Resources/Strings.resx

```diff
@@ -125,102 +125,14 @@
     <value>More app bar</value>
     <comment>332-5305</comment>
   </data>
-  <data name="BadgeIcon" xml:space="preserve">
```

Comment:
We should also remove these from the other Strings.*.resx files.

Reply:
Yeah, I'm aware of that. I was just waiting for your approval.
I will do that with the next commit.